pipelines/install_pipeline.yml

```diff
@@ -1,41 +1,47 @@
 - hosts: localhost
   vars_files:
-    - vars/katello_base.yml
-    - vars/katello_{{ katello_version }}.yml
+    - vars/{{ pipeline_type }}_base.yml
   roles:
     - forklift
 
 - hosts:
-  - pipeline-katello-{{ katello_version }}-centos7
-  - pipeline-proxy-{{ katello_version }}-centos7
+  - pipeline-{{ pipeline_type }}-{{ pipeline_version }}-{{ pipeline_os }}
```

Comment:
Is this now:
```suggestion
  - {{ forklift_name }}
```

Reply:
potentially, it is, but then, who was there first, the chicken or the egg?
(`vars_files` is not loaded at the time ansible evaluates `hosts`)